### PR TITLE
Fix release github action

### LINKED ISSUE #10: Automate compilation of binaries and releases
## The Problem

I only have access to Windows and Linux platforms.
Rusty Autoclicker should be compatible with macOS but I am unable to compile the  macOS binary.
I would have to have all 3 platform binaries in the releases.

Is it possible to automate this process via GitHub actions artifacts?
Ideally the action would produce a draft release that maintainers/collaborators can release manually.

### PR DESCRIPTION
**Pull request by MrTanoshii**
Fixes #10 

Seems to work
https://github.com/MrTanoshii/rusty-autoclicker/actions/runs/3451736545

### CODE REVIEW

**Comment by MrTanoshii** [comment]
@nozwock please review if you're available~

File: .github/workflows/build_artifacts.yml [hunk 1]
```diff
@@ -62,12 +63,21 @@ jobs:
         if: matrix.os != 'windows-latest'
         run: mv target/release/rusty-autoclicker bin/rusty-autoclicker-${{ matrix.build_target }}
 
-      - name: Tarball Linux/MacOS binary
-        if: matrix.os != 'windows-latest'
-        run: cd bin/ && tar -czf rusty-autoclicker-${{ matrix.build_target }}{.tar.gz,} --remove-files
```
[the remark this answers is not in the document]
Comment (nozwock) [reply .github/workflows/build_artifacts.yml:67]: Ig better without `&&`'s

```yaml
        run: |
          cd bin/
          tar -czf rusty-autoclicker-${{ matrix.build_target }}{.tar.gz,}
          rm bin/rusty-autoclicker-${{ matrix.build_target }}
```

[the remark this answers is not in the document]
Comment (MrTanoshii) [reply .github/workflows/build_artifacts.yml:67]: Yup, merged it with the renaming while I was at it

**Review by nozwock — COMMENTED** [review]
Looks great

**Comment by nozwock** [comment]
You should just merge, I can't approve PRs
Screenshot from 2022-11-12 23-31-27

**Comment by nozwock** [comment]
Was introduced by #30

**Pull request merged**